lianad/src/config.rs

```diff
@@ -128,6 +128,19 @@ pub struct ElectrumConfig {
     /// Include "ssl://" for SSL. otherwise TCP will be assumed.
     /// Can optionally prefix with "tcp://".
     pub addr: String,
+    /// If validate_domain == false, domain of ssl certificate will not be validated
+    /// (useful to allow usage of self signed certificates on local network)
+    #[serde(default)]
```

Comment:
This default doesn't seem to be working for me. If my config doesn't contain validate_domain, the value is set to false.

Reply:
It's probably worth adding some unit tests for the `ElectrumConfig` parsing to complement the other config parsing tests.

Reply:
fixed & added tests, the issue was serde use default of field type (false) and not default of the struct, my bad